### PR TITLE
Kafka 2902 streaming config use get base consumer configs

### PR DESCRIPTION
**Pull request by bbejeck**
Changes made for using getBaseConsumerConfigs from StreamingConfig.getConsumerConfigs.

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/StreamingConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.Properties;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+
+/**
+ * Created by bbejeck on 11/27/15.
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/StreamingConfigTest.java]: Could you remove these comments?

**Comment by guozhangwang** [comment]
LGTM, just one minor comment.

**Comment by guozhangwang** [comment]
Hi @bbejeck , do you have time to address the minor comment, I am trying to merge this patch asap.

**Comment by bbejeck** [comment]
Hi @guozhangwang 

I thought I did that already, on last commit I removed the comments from the test.  Is there something else?

**Comment by guozhangwang** [comment]
You are right. Thanks! Merged to trunk.

**Comment by bbejeck** [comment]
Great, Thanks!

**Comment by b-goyal** [comment]
Changes made for using getBaseConsumerConfigs from StreamingConfig.getConsumerConfigs.

**Pull request closed without merging**